decision-tree.yaml

```diff
@@ -923,15 +962,21 @@ conclusions:
       Wel moet deze AI-technologie uiteraard voldoen aan algemene veiligheidsstandaarden."
   - conclusionId: "13.2.1"
     conclusion: "Je bent een gebruiksverantwoordelijke van een AI-model voor algemene doeleinden. Er is sprake van een systeemrisico."
-    obligation: Als gebruiksverantwoordelijke van een AI-model voor algemene doeleinden gelden er geen verplichtingen vanuit het systeemrisico.
+    obligation: "Als gebruiksverantwoordelijke van een AI-model voor algemene doeleinden gelden er geen verplichtingen vanuit het systeemrisico.
+      Het is echter belangrijk om te realiseren dat als een GPAI-model verder wordt ontwikkeld, er mogelijk minimale transparantieverplichtingen van toepassing zijn.
```

Comment:
Maybe change GPAI-model here into 'AI-model voor algemene doeleinden' for consistency?